docs/book/known_issues.md

```diff
@@ -69,3 +70,8 @@ Issue 8<a id="issue_8"></a>: Migrated in-tree vSphere volume deleted by in-tree
 
 - Impact: Migrated in-tree vSphere volumes deleted by in-tree vSphere plugin remains on the CNS UI.
 - Workaround: Admin needs to manually reconcile discrepancies in the Managed Virtual Disk Catalog. Admin needs to follow this [KB article](https://kb.vmware.com/s/article/2147750)
+
+Issue 9<a id="issue_9"></a>: CnsRegisterVolume should prevent importing block volumes that are already available in Pacific created via dynamic provisioning
```

Comment:
How about this?

Issue: CnsRegisterVolume API does not validate if the volume to import is already imported or already present in the supervisor cluster. 
Impact: One of the PVC using the CNS volume will be usable at any point in time. Usage of any other PVC will lead to attach failures.
Workaround: None